src/game/draw.c

```diff
@@ -79,6 +90,7 @@ void GetRoomBounds(int16_t room_num)
         }
     }
     phd_PopMatrix();
+    m_RoomNumStack[m_StackIdx--] = 0;
```

Comment:
I think `m_StackIdx--` alone would suffice, it's not like we'll be trying to read it beyond.
Likewise `phd_PopMatrix` doesn't zero the matrix that was popped from the stack.